### ISSUE TITLE
Solution and project files for Visual Studio on Windows

### ISSUE THREAD

**Issue by martin-braun** [dashpay/dash#4496]
Bitcoin Core offers [solution and project files for Visual Studio](https://github.com/bitcoin/bitcoin/blob/master/build_msvc/README.md) on Windows, now. 

It would be nice if Dash Core could pull those and make them work, so the build process on Windows gets simpler.

**Comment by PastaPastaPasta** [comment]
Yes indeed :) This is on our radar, but it's not trivial to backport, and none of us use windows for development. If you (or anyone else) would like to work on it, we'd appreciate the PR, and we do have a bounty program for submissions. Specifically, bitcoin#11526 is the PR that implemented this for bitcoin

**Comment by martin-braun** [comment]
@PastaPastaPasta Thank you for your quick reply. I just realized that you provide CMakeLists.txt files. Afaik CMake generates solution and project files on Windows. Bitcoin Core doesn't support CMake, so it would make sense for them to commit solution and project files. Is my thinking correct that it should already be possible to generate those files for the Dash Core using CMake?

**Comment by PastaPastaPasta** [comment]
Our CMake file is not operational, it's really just to enable code complete features in ides like CLion (although one of our devs is looking into actually implementing CMake support)

**Comment by martin-braun** [comment]
@PastaPastaPasta Thank you for this piece of information, very useful to me. I think the best route is to bring CMake support to Dash Core, because it will solve this issue as well. Providing project and solution files manually is redundant when CMake serves the same and more.
